app/containers/MassEnergizeSuperAdmin/Community/EditCommunityByCommunityAdmin.js

```diff
@@ -1,432 +1,41 @@
-// TODO: eliminate nearly identical code
-import React, { Component } from "react";
-import PropTypes from "prop-types";
-import { withStyles } from "@material-ui/core/styles";
-import states from "dan-api/data/states";
-import MassEnergizeForm from "../_FormGenerator";
-import { apiCall } from "../../../utils/messenger";
-import { getMoreInfo, groupSocialMediaFields } from "./utils";
-import { connect } from "react-redux";
-import Loading from "dan-components/Loading";
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import brand from 'dan-api/dummy/brand';
+import { PapperBlock } from 'dan-components';
+import CommunityOnboardingForm from './CommunityOnboardingForm';
+import EditCommunityForm from './EditCommunityForm';
 
-const styles = (theme) => ({
-  root: {
-    flexGrow: 1,
-    padding: 30,
-  },
-  field: {
-    width: "100%",
-    marginBottom: 20,
-  },
-  fieldBasic: {
-    width: "100%",
-    marginBottom: 20,
-    marginTop: 10,
-  },
-  inlineWrap: {
-    display: "flex",
-    flexDirection: "row",
-  },
-  buttonInit: {
-    margin: theme.spacing.unit * 4,
-    textAlign: "center",
-  },
-});
-
-class EditCommunityByCommunityAdmin extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      formJson: null,
-      community: undefined,
-      loading: true,
-    };
-  }
-
-  static getDerivedStateFromProps(props, state) {
-    const { communities, match } = props;
-    const { id } = match.params;
-    if (state.community === undefined) {
-      const community = (communities || []).find((c) => c.id.toString() === id);
-      const formJson = createFormJson(community);
-      return { community, formJson };
-    }
-    return null;
-  }
-  async componentDidMount() {
-    const { id } = this.props.match.params;
-    const communityResponse = await apiCall("/communities.info", {
-      community_id: id,
-    });
-    if (communityResponse && !communityResponse.success) {
-      return;
-    }
-
-    const community = communityResponse.data;
-    await this.setStateAsync({ community });
-
-    const formJson =  createFormJson(community);
-    await this.setStateAsync({ formJson });
-  }
-
-  setStateAsync(state) {
-    return new Promise((resolve) => {
-      this.setState(state, resolve);
-    });
-  }
+class EditCommunityByCommunityAdmin extends React.Component {
 
   render() {
-    const { classes } = this.props;
-    const { formJson, community } = this.state;
-    if (!community || !formJson) return <Loading />;
+    const description = brand.desc;
+    console.log(this.props)
```

Comment:
is there a reason for this log statement?

Reply:
I'll remove it when I merge it. Good catch , BTW. I missed it when I started reviewing this on Monday.